subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java

```diff
@@ -171,10 +173,12 @@ private boolean isTeamCityAgent(String line) {
 
     private static StdoutAndPatterns ps() {
         String[] command = isWindows() ? new String[]{"wmic", "process", "get", "processid,commandline"} : new String[]{"ps", "x"};
-        ExecResult result = run(command).assertZeroExit();
+        ExecResult result = run(command);
+        output.printf("Run: " + Arrays.toString(command));
+        output.printf("Stdout: " + result.stdout);
```

Comment:
Are we still printing out the stdout here?

Reply:
No, the `output` is a file: `output = new PrintStream(outputFile)`.